nss-day-cohort-29/nutshell-sunset-chaserz
# Allisonpractice

# Description

Added forms.js with a form for Tasks that appends to the DOM. forms.js coordinates with main.js to import/export relevant elements.


## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

Identify forms.js in scripts directory. Refresh browser and see if the form appears.

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings
- [ x] I have added test instructions that prove my fix is effective or that my feature works
